### PR TITLE
docs:fix unhealthy.failures attribute description of api-breaker plugin

### PR DESCRIPTION
**Pull request by haynesxiang**
### Description

Fixes # (issue)

### Checklist

- [ ] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/en/latest/plugins/api-breaker.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ In an unhealthy state, if the Upstream service responds with a status code from
...
 | unhealthy.http_statuses | array[integer] | False    | [500]   | [500, ..., 599] | Status codes of Upstream to be considered unhealthy.                                                                                                                                                                                         |
-| unhealthy.failures      | integer        | False    | 3       | >=1             | Number of consecutive failures for the Upstream service to be considered unhealthy.                                                                                                                                                          |
+| unhealthy.failures      | integer        | False    | 3       | >=1             | Number of failures for the Upstream service to be considered unhealthy，which only takes effect if the number of consecutive success requests between failed requests does not reach `healthy.successes`.                                                                                                                                                          |
```
Comment (tokers) [inline docs/en/latest/plugins/api-breaker.md:49]: The description is confusing. Maybe we don't need to highlight it.

@tzssangglass What's your opinion?

**Review by tzssangglass — DISMISSED** [review]

File: docs/en/latest/plugins/api-breaker.md (hunk 1 above)
Comment (tzssangglass) [reply docs/en/latest/plugins/api-breaker.md:49]: It is a bit confusing

Comment (tzssangglass) [reply docs/en/latest/plugins/api-breaker.md:49]: what about:  If the upstream service returns `unhealthy.http_statuses` `unhealthy.failures` times in a row within a certain period of time, it is considered unhealthy.

File: docs/zh/latest/plugins/api-breaker.md [hunk 2]
```diff
@@ -48,14 +48,14 @@ title: api-breaker
...
-| unhealthy.failures      | integer        | 可选   | 3          | >=1             | 触发不健康状态的连续错误请求次数 |
+| unhealthy.failures      | integer        | 可选   | 3          | >=1             | 触发不健康状态的异常请求次数，并且异常请求间连续正常请求未达到触发健康状态的次数 |
 | healthy.http_statuses   | array[integer] | 可选   | {200}      | [200, ..., 499] | 健康时候的状态码                 |
 | healthy.successes       | integer        | 可选   | 3          | >=1             | 触发健康状态的连续正常请求次数   |
 
 ## 启用方式
 
 这是一个示例，在指定的路由上启用 `api-breaker` 插件。
-应答 500 或 503 连续 3 次，触发熔断。应答 200 连续 1 次，恢复健康。
+应答 500 或 503 3 次且异常请求间应答 200 未达到1次，触发熔断。应答 200 连续 1 次，恢复健康。
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/api-breaker.md:58]:
```suggestion
应答 500 或 503 的次数达到 3 且异常请求间应答 200 未达到 1 次，触发熔断。应答 200 连续 1 次，恢复健康。
```

File: docs/zh/latest/plugins/api-breaker.md [hunk 3]
```diff
@@ -85,7 +85,7 @@ curl "http://127.0.0.1:9080/apisix/admin/routes/1" -H 'X-API-KEY: edd1c9f034335f
 
 ## 测试插件
 
-使用上游的配置，如果你的上流服务返回 500，连续 3 次。客户端将会收到 502（break_response_code）应答。
+使用上游的配置，如果你的上流服务返回 500 3 次且异常请求间返回 200 未达到1次。客户端将会收到 502（break_response_code）应答。
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/api-breaker.md:88]:
```suggestion
使用上游的配置，如果你的上流服务返回 500 次数达到 3 且异常请求间返回 200 未达到 1 次。客户端将会收到 502（break_response_code）应答。
```

File: docs/en/latest/plugins/api-breaker.md (hunk 1 above)
Comment (haynesxiang) [reply docs/en/latest/plugins/api-breaker.md:49]: if healthy.successes is configured with 2 and unhealthy.failures is configured wituh 3, there are scenarios:
1.request 1 return failed, request 2 return failed,  request 3 return success, request 4 return failed,  it is considered unhealthy.
2.request 1 return failed, request 2 return failed,  request 3 return success, request 4 return success, request 4 return failed,  it is considered as healthy status.

So, how about describe it:  If the upstream service returns unhealthy.http_statuses unhealthy.failures times and returns healthy.http_statuses times does not reach healthy.successes within a certain period of time, it is considered unhealthy.

Or we just describe it as "Number of failures for the Upstream service to be considered unhealthy within a certain period of time"

**Review by tokers — APPROVED** [review]

File: docs/zh/latest/plugins/api-breaker.md [hunk 4]
```diff
@@ -48,14 +48,14 @@ title: api-breaker
 | break_response_headers  | array[object]  | 可选   | 无         | [{"key":"header_name","value":"can contain Nginx $var"}] | 不健康返回报文头，这里可以设置多个。该字段仅在 `break_response_body` 被配置时生效。这个值能够以 `$var` 的格式包含 APISIX 变量，比如 `{"key":"X-Client-Addr","value":"$remote_addr:$remote_port"}`。 |
 | max_breaker_sec         | integer        | 可选   | 300        | >=3             | 最大熔断持续时间                 |
 | unhealthy.http_statuses | array[integer] | 可选   | {500}      | [500, ..., 599] | 不健康时候的状态码               |
-| unhealthy.failures      | integer        | 可选   | 3          | >=1             | 触发不健康状态的连续错误请求次数 |
+| unhealthy.failures      | integer        | 可选   | 3          | >=1             | 在一定时间内触发不健康状态的异常请求次数|
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/api-breaker.md:51]:
```suggestion
| unhealthy.failures      | integer        | 可选   | 3          | >=1             | 在一定时间内触发不健康状态的异常请求次数 |
```

Comment (SylviaBABY) [reply docs/zh/latest/plugins/api-breaker.md:51]: image

Comment (haynesxiang) [reply docs/zh/latest/plugins/api-breaker.md:51]: ok, thanks

**Review by SylviaBABY — APPROVED** [review]

**Review by tokers — APPROVED** [review]

**Pull request merged**